ecosystem/sep-0024.md

```diff
@@ -190,6 +191,72 @@ The deposit flow can only be fulfilled if the Stellar `Account` has established
 5. `Anchor` detects the trust line creation in the `Account`. If the asset is `AUTH_REQUIRED`, `Anchor` approves the new trust line.
 6. `Anchor` proceeds with the deposit flow.
 
+##### Claimable Balances
+
+[Claimable Balances](https://www.stellar.org/blog/protocol-14-improvements) allows payments to accounts that do not have a trust-line for the asset sent.
+
+**Wallets** poll for their users' received Claimable Balances so that users can "claim" them.
+
+**Anchors** can optionally use this functionality to send SEP-24 deposit transactions to Stellar accounts that don't yet have a trust-line to the asset they have requested.
+
+**Wallet Claimable Balance Flow**
+
+Wallets will still follow the [typical deposit flow](#Basic-Wallet-Implementation).
+The POST request however will extend to include a `claimable_balance` parameter
+
+```
+POST TRANSFER_SERVER_SEP0024/transactions/deposit/interactive
+Content-Type: multipart/form-data
+```
+
+New Request Parameter:
+
+Name | Type | Description
+-----|------|------------
+`claimable_balance` | boolean | True if submitting transactions as a claimable balance is accepted, false otherwise.
+
+**Wallet Claimable Balance Polling**
+
+The wallet must either listen for a callback from the anchor with the
+`claimable_balance_id` or poll Horizon's
+[`/claimable_balances/`](https://developers.stellar.org/api/resources/claimablebalances/list/)
+endpoint to pull any available claimable balances for the claimant.
+
+*Recommended request*
+`GET /claimable_balances?claimant={:claimant}&order={asc,desc}&limit={1-200}`
+
+This allows the wallet to update the claimant that their is a balance available
+to claim and they may need to establish a [trust-line](#stellar-account-doesnt-trust-asset) in order to do so.
+
+**Anchor Claimable Balance Flow**
+
+1. The given Stellar `account` does not have a trust-line to the asset's issuer account.
+2. The `Anchor` proceeds with the deposit flow via the [CAP-0023](../core/cap-0023.md) Create Claimable Balance operation.
+
+**Anchor Claimable Balance Field Changes**
+
+### Fields for deposit transactions
+
+Name | Type | Description
+-----|------|------------
+`claimable_balance_id` | string | (optional) ClaimableBalanceID is a required parameter when claiming a Claimable Balance operation.
```

Comment:
Same thing here, this needs to go in the actual response from the `POST /interactive` endpoints and referenced here.

Reply:
Now that I'm thinking about this more, I'm realizing it's impossible for anchors to return a `claimable_balance_id` for a payment that hasn't been submitted to the network at this point. So I don't think we can have this response attribute at all.